platform/pulp/platform/fields.py

```diff
@@ -0,0 +1,80 @@
+"""
```

Comment:
My comment wasn't well written. Yes fields.py and models.py should be separate. Where do Django app layouts recommend the fields.py live?